app/tailwind.css

```diff
@@ -272,31 +216,10 @@ html {
 }
 
 ::-webkit-scrollbar-thumb {
-  background: var(--color-primary);
+  background: #e57a21;
   border-radius: 4px;
 }
 
 ::-webkit-scrollbar-thumb:hover {
   background: #ee842c;
 }
-
-/* Loading skeleton animation */
-@keyframes skeleton-loading {
-  0% {
-    background-position: -200px 0;
-  }
-  100% {
-    background-position: calc(200px + 100%) 0;
-  }
-}
-
-.skeleton {
-  background: linear-gradient(
-    90deg,
-    transparent,
-    rgba(255, 255, 255, 0.1),
-    transparent
-  );
-  background-size: 200px 100%;
-  animation: skeleton-loading 1.5s infinite;
-}
```

Comment:
why this was here? I never undestood where this came from

Reply:
this was for showing the shimmer when component loads.